agent/assets/ztp/registries_conf.j2

```diff
@@ -1,28 +1,7 @@
+{% if mirror_images == "true" %}
 {# base oc mirror registries.conf #}
+{{ registries }}
 {% if mirror_command == "oc-mirror" %}
```

Comment:
The two `if`s can be combined into one. Otherwise, LGTM.

Reply:
Fixed